### DISCUSSION TITLE
integration DELTA MAX 1600

### DISCUSSION THREAD

**Discussion by majincyril** [vwt12eh8/hassio-ecoflow#95] [Q&A]
Good morning,

 I have a problem integrating my DELTA MAX in Home assistant 
 I did the integration via HACS but when I enter the iP of my delta Max, it runs in a loop and nothing happens:
 My Home assistant version: 2023.8.3
 Can you help me ?  Have you ever had this problem ?

**Comment by Cricri6799** [comment]
> Good morning,
> 
>  I have a problem integrating my DELTA MAX in Home assistant 
>  I did the integration via HACS but when I enter the iP of my delta Max, it runs in a loop and nothing happens:
>  My Home assistant version: 2023.8.3
>  Can you help me ?  Have you ever had this problem ?
> 

Salut Cyril, essaye plutôt l'api : https://github.com/faken/hassio-ecoflow-cloud
